code/modules/spells/spell_types/tower_of_babel.dm

```diff
@@ -0,0 +1,83 @@
+GLOBAL_DATUM(tower_of_babel, /datum/tower_of_babel)
+
+/datum/tower_of_babel
+
+/datum/tower_of_babel/New(mob/badmin)
+	if(badmin)
+		to_chat(badmin, span_warning("You have stricken the station with the Tower of Babel!"))
```

Comment:
I don't think this line is necessary, since the admin will also be getting the message_admins line

Reply:
Removed with b5b6e01648434e669e991716e517833ac16c936d